lms/static/scripts/frontend_apps/components/BookPicker.js

```diff
@@ -71,8 +71,13 @@ export default function BookPicker({ onCancel, onSelectBook }) {
   const canSubmit =
     (step === 'select-book' && book) || (step === 'select-chapter' && chapter);
 
+  // Opt out of Modal's automatic focus handling; route focus manually in
+  // sub-components
+  const initialFocus = /** @type null */ (null);
+
   return (
     <Modal
+      initialFocus={initialFocus}
```

Comment:
```suggestion
      // Opt out of Modal's automatic focus handling; route focus manually in
      // sub-components
      initialFocus={null}
```

This also works to avoid the dummy variable.

Reply:
Yes, nicer. Thank you; I've made this change.